docs/Tutorials/Visualization.md

```diff
@@ -2,26 +2,75 @@
 Distributed under the MIT License.
 See LICENSE.txt for details.
 \endcond
-# Visualization {#tutorial_visualization}
+# Running and Visualizing {#tutorial_visualization}
 
-### 1D %Data Volume %Data
+### Building an Executable from an Existing Source File
 
-Volume data from one dimensional simulations is visualized using the python
-script `Support/bin/Render1dData.py`. Run
-`python /path/to/Render1dData.py --help` to view the help text for the script.
-Let us consider a Burgers evolution where we want to visualize the solution,
-`U`, then in the run directory you can use
+SpECTRE source files for executables are located in
+`src/Evolution/Executables` and can be compiled by running the command
+`make EXECUTABLE_NAME` where `EXECUTABLE_NAME` is the name of the executable
+as defined in the `CMakeLists.txt` file located in the same directory as the
+source file. For example, to compile the executable that evolves a scalar wave
+using a three-dimensional domain, one runs the command:
+`make EvolveScalarWave3D`, which then results in an executable of the same name
+in the `bin` directory of the user's build directory.
 
-### 2D & 3D %Data Volume %Data In ParaView
+### Running an Evolution Executable
 
-Once you have generated volume data you must run the
-executable `ApplyObservers` in the run directory to generate an XDMF
-(`.xmf` extension) file. Open the `VisVolumeData.xmf` file in ParaView
-and selet the `Xdmf Reader`, *not* the version 3 readers. On the left hand side
-of the main ParaView window is a section named `Properties`, here you must
-click the highlighted `Apply` button. ParaView will now render your volume data.
+Each SpECTRE executable is paired with a user-provided YAML file that
+specifies the runtime settings of the evolution. This is where information on
+options such as the Domain, AnalyticSolution, TimeStepper, etc is supplied.
+Example input files are kept in `tests/InputFiles`, and are written to provide
+a functional base input file which the user can then modify as desired.
+The YAML file is passed as an argument to the executable using the flag
+`--input-file`. For example, for a scalar wave evolution, run the command:
+`./EvolveScalarWave3D --input-file Input3DPeriodic.yaml`, where
+`Input3DPeriodic.yaml` was copied over from the `InputFiles` directory to the
+`bin` directory. A successful evolution will result in the creation of two H5
+files whose names can be specified in the YAML file under the options
+`VolumeFileName` and `ReductionFileName`. Visualization of the volume data will
```

Comment:
This does not match the current behavior.  The example input file does not produce any output.